lore/admins/reboot-zuul.md

```diff
@@ -0,0 +1,83 @@
++-----+----------+------+---------------+--------+---------+---------------------------+---------------------------+--------------------------------------+--------------+--------+-------------+---------+
+| 359 | cicloud  | None | ubuntu-xenial | zuul   | zuul    | ubuntu-xenial-cicloud-359 | ubuntu-xenial-cicloud-359 | 4d3f7503-64f0-4c6b-8066-66ec64cace2b | 192.168.0.64 | delete | 00:00:00:00 | None    |
++-----+----------+------+---------------+--------+---------+---------------------------+---------------------------+--------------------------------------+--------------+--------+-------------+---------+
+(nodepool) nodepool@nodepool:/home/ubuntu$ nodepool delete 360
++-----+----------+------+---------------+--------+---------+---------------------------+---------------------------+--------------------------------------+--------------+--------+-------------+---------+
+| ID  | Provider | AZ   | Label         | Target | Manager | Hostname                  | NodeName                  | Server ID                            | IP           | State  | Age         | Comment |
++-----+----------+------+---------------+--------+---------+---------------------------+---------------------------+--------------------------------------+--------------+--------+-------------+---------+
+| 360 | cicloud  | None | ubuntu-xenial | zuul   | zuul    | ubuntu-xenial-cicloud-360 | ubuntu-xenial-cicloud-360 | bca6c0e6-e1a0-4894-a550-ef1fef302819 | 192.168.0.65 | delete | 00:00:00:00 | None    |
++-----+----------+------+---------------+--------+---------+---------------------------+---------------------------+--------------------------------------+--------------+--------+-------------+---------+
+```
+
+And then kill the hoist node:
+
+```text
+(nodepool) nodepool@nodepool:/home/ubuntu$ nodepool delete 361
++-----+----------+------+--------------+--------+---------+--------------------------+--------------------------+--------------------------------------+--------------+--------+-------------+---------+
+| ID  | Provider | AZ   | Label        | Target | Manager | Hostname                 | NodeName                 | Server ID                            | IP           | State  | Age         | Comment |
++-----+----------+------+--------------+--------+---------+--------------------------+--------------------------+--------------------------------------+--------------+--------+-------------+---------+
+| 361 | cicloud  | None | ubuntu-hoist | zuul   | zuul    | ubuntu-hoist-cicloud-361 | ubuntu-hoist-cicloud-361 | 00ced614-3fec-4eb9-95c5-8c77d0cd32e5 | 192.168.0.66 | delete | 00:00:00:00 | None    |
++-----+----------+------+--------------+--------+---------+--------------------------+--------------------------+--------------------------------------+--------------+--------+-------------+---------+
+```
```

Comment:
Should add a trailing blurb to watch node pool list to make sure node pool spins up new hosts to replace the deleted ones.